client/src/sge/GraphicsShaders.cpp

```diff
@@ -37,7 +37,17 @@ std::string sge::readShaderSource(std::string filename) {
  */
 void sge::initShaders()
 {
-    std::string vertexShaderSource = readShaderSource("./shaders/static.vert.glsl");
+    std::string prefix;
+
+    #if defined(_WIN32)
+        std::string anh_prefix = "D:/UCSD/2024/Spring/125/group5/client";
```

Comment:
Don't include your name in the variable name

Reply:
I made that variable so that if @Melody-Ruth or @benjxia need to add your own path it won't be a problem.

Reply:
and tbh I can't think of better name for the variable

Reply:
clientWorkingDirectoryPrefix, a little verbose but it gets the point across

Reply:
I put the variable in a header file now - one single location

Reply:
Does the relative path not work for you? It's worked fine for me to just say "models/[whatever model you want]", and run the executable from the client folder. Hardcoding absolute filepaths seems like a bad idea to me unless it's necessary to make things work (since that limits which machines can run the code and means different machines will be running different code).

Reply:
This is the old commit. Check out my new commit. I move the prefix to one single header file. To answer your question: if I did that it works but it's like 2-3 steps and I have to do that every time I want to run the program. Here you just have to change one time and you can always change back to relative if you want

Reply:
to be honest I still don't see how this change is completely necessary as we've all been assuming that client runs from a particular path so far

Reply:
it's likely far easier to run client from a specific directory than needing to modify the source and recompile for every machine we run on

Reply:
Hm I guess that make sense. I will close this PR